packages/client-sdk-web/src/internal/data-client.ts

```diff
@@ -1331,12 +1331,13 @@ export class DataClient<
   private async sendDictionaryGetFields(
     cacheName: string,
     dictionaryName: string,
-    fields: string[]
+    fields: string[] | Uint8Array[]
   ): Promise<CacheDictionaryGetFields.Response> {
     const request = new _DictionaryGetRequest();
     request.setDictionaryName(dictionaryName);
-    request.setFieldsList(fields);
+    request.setFieldsList(this.convertArrayToB64Strings(fields));
```

Comment:
here - we need to convert the fields to b64 for the Request, but when we are constructing the response, we need to retain the un-encoded versions of them.